example/src/main/res/drawable/ic_rewards_nav_icon.xml

```diff
@@ -0,0 +1,9 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Should this icon be committed? As a visual asset, there are copyright issues with releasing this into the public domain if we do not have the explicit permission to do so.

Reply:
Rather use a material icon from Android studio. You can find them in the Vector Asset importer under clipart:
![image](https://user-images.githubusercontent.com/16672702/108478467-359f5b00-729d-11eb-98c3-9e567e25c2a3.png)